ui/raidboss/data/06-ew/raid/p10n.ts

```diff
@@ -1,14 +1,201 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  silkspitOnYou: boolean;
+}
+
+const bossNameUnicode = 'Pand\u00e6monium';
```

Comment:
I'm not sure I understand why you can't use `Pandæmonium` directly everywhere.  What sort of hiccup did that cause?

Reply:
The 'æ' character did not display correctly in the timeline or in raidemulator when used in a trigger `id` or timeline entry name (it displayed as a diamond with a question-mark in both cases).  During some testing, I also found that timeline entries would not sync to log lines where that character was in the regex when I intentionally adjusted some timeline entries to try to test syncs.  It also seemed that certain triggers were not firing correctly, although once I made the switch and things started working, I admittedly didn't dig in deeper to find root cause.